src/main/groovy/org/mockito/release/notes/model/Commit.java

```diff
@@ -8,6 +8,11 @@
 public interface Commit {
 
     /**
+     * Commit identifier. For git it would be 'hash' (SHA-1)
+     */
+    String getCommitId();
```

Comment:
I found usage for commitId, please see `GitHubContributorsFetcher.cantFindProfileForAuthorError(...)`